package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "chat-frontend",
+  "version": "0.1.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "dotenv": {
+      "version": "4.0.0",
+      "resolved": "https://registry.npmjs.org/dotenv/-/dotenv-4.0.0.tgz",
+      "integrity": "sha1-hk7xN5rO1Vzm+V3r7NzhefegzR0="
+    }
+  }
+}
```

Comment:
Any ideas where this file came from? I've seen it generated when the `npm install` command is used instead of `yarn install`. 

Let's remove it, unless there's a real reason it's needed here.